src/components/map/mapControls/MapLayersControl.tsx

```diff
@@ -80,32 +84,33 @@ class MapLayersControl extends React.Component
                     />
                     <NetworkDateControl selectDate={this.selectDate}/>
                     <div className={s.sectionDivider} />
+                    <div className={s.inputTitle}>GEOMETRIAT</div>
                     <div className={s.checkboxContainer}>
                         <Checkbox
                             onClick={this.toggleMapLayerVisibility(MapLayer.link)}
                             checked={NetworkStore.isMapLayerVisible(MapLayer.link)}
-                            text={'Näytä alueen linkit'}
+                            text={'Alueen linkit'}
```

Comment:
Set
text={'Alueen linkit'}
=>
text='Alueen linkit'
on all similar places in this file, there are plenty

Reply:
Refactored. I guess that is our convention from now on through the app